### PR TITLE
Add new abraham parameters for pure solvents and co-solvents

### PR DESCRIPTION
**Pull request by oscarwumit**
1. Add MeOH, MeOH:Water (50:50), Acetonitrile:Water(40:60, 60:40)
abraham solvent parameters to solvent.py
2. Add a citation section at the bottom of solvent.py for future
reference

### CODE REVIEW

**Comment by alongd** [comment]
@oscarwumit, can you rebase this branch? Let's resolve the tests issues and get it merged

**Comment by oscarwumit** [comment]
ok

**Comment by oscarwumit** [comment]
@alongd would you please review this pr? thanks :-)

File: input/solvation/libraries/solvent.py [hunk 1]
```diff
@@ -7,6 +7,13 @@
 Abraham solvent parameters (s_g, b_g, e_g, l_g, a_g, c_g) are from:
 Jalan, Amrit. (2014). Predictive kinetic modeling of low-temperature hydrocarbon oxidation (Doctoral dissertation).
 Cambridge, MA: Massachusetts Institute of Technology.
+
+Reference legend:
+[Abraham2012] The hydrogen bond properties of water from 273 K to 573 K; equations for the prediction of gas-water partition coefficients Michael H. Abraham and William E. Acree Jr Phys. Chem. Chem. Phys., 2012,14, 7433–7440
+[Abraham2016] Equations for the Partition of Neutral Molecules, Ionsand Ionic Species from Water to Water–MethanolMixtures Michael H. Abraham and William E. Acree Jr J Solution Chem (2016) 45:861–874
+[Gagliardi2007] Static Dielectric Constants of Acetonitrile/Water Mixtures at Different Temperatures and Debye−Hückel A and a0B Parameters for Activity Coefficients Leonardo G. Gagliardi,†, Cecilia B. Castells,‡, Clara Ràfols,†, Martí Rosés,† and, and Elisabeth Bosch*,† Journal of Chemical & Engineering Data 2007 52 (3), 1103-1107 DOI: 10.1021/je700055p
```
Comment (alongd) [inline input/solvation/libraries/solvent.py]: There are non-ASCII characters here. Could you remove/replace them?
See discussion here: https://github.com/ReactionMechanismGenerator/RMG-Py/issues/1144

**Comment by oscarwumit** [comment]
i removed them. would you mind taking another look? thanks @alongd

File: input/solvation/libraries/solvent.py [hunk 2]
```diff
@@ -7,6 +7,13 @@
 Abraham solvent parameters (s_g, b_g, e_g, l_g, a_g, c_g) are from:
 Jalan, Amrit. (2014). Predictive kinetic modeling of low-temperature hydrocarbon oxidation (Doctoral dissertation).
 Cambridge, MA: Massachusetts Institute of Technology.
+
+Reference legend:
+[Abraham2012] The hydrogen bond properties of water from 273 K to 573 K; equations for the prediction of gas-water partition coefficients Michael H. Abraham and William E. Acree Jr Phys. Chem. Chem. Phys., 2012,14, 7433–7440
+[Abraham2016] Equations for the Partition of Neutral Molecules, Ionsand Ionic Species from Water to Water–MethanolMixtures Michael H. Abraham and William E. Acree Jr J Solution Chem (2016) 45:861–874
+[Gagliardi2007] Static Dielectric Constants of Acetonitrile/Water Mixtures at Different Temperatures and Debye−Hückel A and a0B Parameters for Activity Coefficients Leonardo G. Gagliardi, Cecilia B. Castells, Clara Rafols, Marti Roses and, and Elisabeth Bosch, Journal of Chemical & Engineering Data 2007 52 (3), 1103-1107 DOI: 10.1021/je700055p
```
Comment (alongd) [inline input/solvation/libraries/solvent.py]: also `Hückel`?

Comment (oscarwumit) [reply input/solvation/libraries/solvent.py]: thanks I missed that

**Comment by oscarwumit** [comment]
please take another look thanks :-) @alongd

**Comment by alongd** [comment]
Thanks!

**Pull request merged**